src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -47,26 +49,63 @@ export class DotHttpErrorManagerService {
      * @memberof DotHttpErrorManagerService
      */
     handle(err: ResponseView): Observable<DotHttpErrorHandled> {
-        const result: DotHttpErrorHandled = {
-            redirected: this.callErrorHandler(err.response.status)
-        };
+        return this.getMessages().switchMap(() => {
+                const result: DotHttpErrorHandled = {
+                    redirected: this.callErrorHandler(err.response)
+                };
 
-        if (err['bodyJsonObject'].error) {
-            result.forbidden = this.contentletIsForbidden(err['bodyJsonObject'].error);
-        }
+                if (err['bodyJsonObject'].error) {
+                    result.forbidden = this.contentletIsForbidden(err['bodyJsonObject'].error);
+                }
 
-        return Observable.of(result);
+                return Observable.of(result);
+            }).take(1);
     }
 
-    private callErrorHandler(code: HttpCode): boolean {
-        const errors = {};
+    get fakeForbiddenError(): ResponseView {
+        const response: Response = new Response({
```

Comment:
I did this utility for the tests: https://github.com/dotCMS/core-web/blob/master/src/app/test/response-view.mock.ts#L4

Should we do something like this for this?